schema/gosling.schema.json

```diff
@@ -3150,7 +3150,7 @@
           "$ref": "#/definitions/AxisPosition",
           "description": "not supported"
         },
-        "xDomain": {
```

Comment:
Can you check updates on this file? Since `gosling.schema.ts` is not edited, I think this file should not be updated as well.

Reply:
Sure!